src/footer/footer.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Col, Container, Row } from "react-bootstrap";
+import "./footer.scss";
+
+export const Footer = () => {
+  return (
+    <Container fluid className="text-center footer">
+      <Row>
+        <Col>
+          <svg
```

Comment:
The code would be cleaner if these SVG came from the `svg.tsx` like on the home page..
What do you think about it?

Reply:
Moved the svg-s into the svg.tsx file.